internal/proto/codec/builtin.go

```diff
@@ -123,6 +123,8 @@ func (codecUtil) NextFrameIsDataStructureEndFrame(frameIterator *proto.ForwardFr
 	return frameIterator.PeekNext().IsEndFrame()
 }
 
+// NextFrameIsNullFrame
+// Deprecated:
 func (codecUtil) NextFrameIsNullFrame(frameIterator *proto.ForwardFrameIterator) bool {
```

Comment:
Is the first line in the comment the name of the alternative or this function? I mean, we say that it is deprecated and I would like to make sure that we are linking the alternative.

Reply:
The doc comment must start with the name of the identifier. I've improved the deprecation notices at: https://github.com/hazelcast/hazelcast-go-client/pull/910/commits/05d7f6629a2d0b144925608c3f3e9e07b5f9a570